setup.py

```diff
@@ -37,7 +37,7 @@
     },
     install_requires=[
         'aiohttp-graphql',
-        'aiohttp<3.6',
+        'aiohttp!=3.6.0',
```

Comment:
Why `!=` ? Are there any problems with this version?

Reply:
Yes, it is.